libs/shared/services/src/lib/api-routes.constant.ts

```diff
@@ -404,6 +404,8 @@ export const API_ROUTES = {
       LIVE_EVENT_SESSIONS: 'api/v2/track_slots/public_live_event_sessions',
```

Comment:
Let's make a proper order
 TRACK_SLOTS_BY_DATE: 'api/v2/track_slots/track_slots_by_date',
    TRACK_SLOTS_BY_DATE_AND_LOCATION: 'api/v2/track_slots/track_slot_by_date_and_location',
    
    this both will be before public object